src/Raven.Client/Documents/Indexes/ExpressionStringBuilder.cs

```diff
@@ -125,19 +125,24 @@ private void Out(string s)
 
         private void OutMember(Expression instance, MemberInfo member, Type exprType)
         {
+            bool isId = false;
             var name = GetPropertyName(member.Name, exprType);
             if (TranslateToDocumentId(instance, member, exprType))
             {
-                name = Constants.Documents.Indexing.Fields.DocumentIdFieldName;
+                isId = true;
+                Out("Id(");
```

Comment:
shouldn't this be lower case and use the constant?

Reply:
No, this is creating a
```Id(document)``` method call
to https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Server/Documents/Indexes/Static/StaticIndexBase.cs#L60